substrafl/remote/register/generate_wheel.py

```diff
@@ -34,8 +34,6 @@ def local_lib_wheels(lib_modules: List, operation_dir: Path, python_major_minor:
     wheels_dir = operation_dir / dest_dir
     wheels_dir.mkdir(exist_ok=True, parents=True)
     for lib_module in lib_modules:
-        wheel_name = f"{lib_module.__name__}-{lib_module.__version__}-py3-none-any.whl"
```

Comment:
What was the use of this line?

Reply:
None, it is redefine just below, that's why I deleted it